java/api/src/main/java/org/ray/api/options/ActorCreationOptions.java

```diff
@@ -13,9 +13,18 @@
 
   public final int maxReconstructions;
 
-  private ActorCreationOptions(Map<String, Double> resources, int maxReconstructions) {
+  public final String workerStartingPrefix;
+
+  public final String workerStartingSuffix;
```

Comment:
I think, at the user api level, we should simply allow passing in the jvm options as a map. 
And under the hood, we can use prefix/suffix to make this feature language-independent.

Reply:
As the result of offline discussions, we'd like to  use `String jvmOptions;` as Java API.